toolchain/mfc/params_tests/test_validate.py

```
@@ -0,0 +1,167 @@
+"""
+
+import unittest
+from ..params.validate import (
+    validate_constraints,
+    check_dependencies,
+    validate_case,
+    format_validation_results,
+)
+from ..params.schema import ParamDef, ParamType, Stage
+from ..params.registry import REGISTRY
+from ..params import definitions  # noqa: F401  Populate registry
```

Ruff (0.14.14)

[warning] Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)